packages/react/src/MeasureReportDisplay/MeasureReportDisplay.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, SimpleGrid, em } from '@mantine/core';
+import { MeasureReport, MeasureReportGroup, Reference } from '@medplum/fhirtypes';
+import { useResource, useSearchOne } from '@medplum/react-hooks';
+import React from 'react';
+import { MeasureReportGroupDisplay, MeasureTitle } from './MeasureReportGroup/MeasureReportGroup';
+import { useMediaQuery } from '@mantine/hooks';
+
+export interface MeasureReportDisplayProps {
+  readonly measureReport: MeasureReport | Reference<MeasureReport>;
+}
+
+export function MeasureReportDisplay(props: MeasureReportDisplayProps): JSX.Element | null {
+  // Relies on window.matchMedia(), will return false if not available https://mantine.dev/hooks/use-media-query/
+  const isMobile = useMediaQuery(`(max-width: ${em(750)})`);
+  const report = useResource(props.measureReport);
+  const [measure] = useSearchOne('Measure', { url: report?.measure });
+
+  if (!report) {
+    return null;
+  }
+
+  return (
+    <Box>
+      {measure && <MeasureTitle measure={measure} />}
+      <SimpleGrid cols={!isMobile ? 3 : 1} spacing={'xs'}>
```

Comment:
I think the recommended way to do it is to use the Mantine `breakpoints` prop: https://v6.mantine.dev/core/simple-grid/#breakpoints

```tsx
import { SimpleGrid } from '@mantine/core';

function Demo() {
  return (
    <SimpleGrid
      cols={4}
      spacing="lg"
      breakpoints={[
        { maxWidth: '62rem', cols: 3, spacing: 'md' },
        { maxWidth: '48rem', cols: 2, spacing: 'sm' },
        { maxWidth: '36rem', cols: 1, spacing: 'sm' },
      ]}
    >
      <div>1</div>
      <div>2</div>
      <div>3</div>
      <div>4</div>
      <div>5</div>
    </SimpleGrid>
  )
}
```